src/rest/server.go

```diff
@@ -0,0 +1,56 @@
+package rest
+
+import (
+	"devicemanager/config"
+	odimConfig "github.com/ODIM-Project/ODIM/lib-utilities/config"
+	"github.com/kataras/iris/v12"
+	"github.com/sirupsen/logrus"
+	"net/http"
+)
+
+func InitializeAndRunApplication(config *config.Config) {
+	app := iris.New()
+	app.WrapRouter(trailingSlashRouter)
+	createRouting(app, config)
+
+	server, err := newHttpServer(config)
+	if err != nil {
+		logrus.Fatal("error during initialization of Device Manager server: " + err.Error())
+	}
+
+	app.Run(iris.Server(server))
+}
+
+func createRouting(app *iris.Application, config *config.Config) {
+	basicAuthHandler := newBasicAuthHandler(config.UserName, config.Password)
+	routes := app.Party("/ODIM/v1")
+	{
+		systems := routes.Party("/Systems", basicAuthHandler)
```

Comment:
Why do we define systems here? The handler for route is missing.

Reply:
By defining `systems` we make code looks more clean and readable. 
We are not expecting anything at `/ODIM/v1` so there is no handler for that.